src/SamplesApp/SamplesApp.Droid/Properties/AndroidManifest.xml

```diff
@@ -2,6 +2,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="uno.platform.unosampleapp" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
```

Comment:
```suggestion
<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="uno.platform.unosamplesapp" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
```

Reply:
Changing this requires changing the name in the UI Tests: 
https://github.com/unoplatform/uno/blob/9b61e3ddf2a62e6ac70a7e0a1b586a6bbb67cbfd/src/SamplesApp/SamplesApp.UITests/Constants.cs#L14